silverback/subscriptions.py

```diff
@@ -124,6 +136,26 @@ async def get_subscription_data(self, sub_id: str) -> AsyncGenerator[dict, None]
             else:
                 yield await queue.get()
 
+    async def get_subscription_data_nowait(
```

Comment:
Is the intended use case for this something like this:

```py
while data := subscriptions.get_subscriptions_data_nowait(sub_id):
    # handle `data`
# no more data in queue, go handle other subs now
```

Reply:
Basically, yes. I don't want to hang on an empty queue.

Reply:
Do you think this refactor might make sense for the built-in SDK websocket runner to use in order to lesson these types of errors we have been seeing there as well?

Reply:
The websocket runner doesn't have the same issue I ran into since it splits off asyncio tasks for each subscription. I don't think changing that for a single Silverback app will provide any improvements.